proto/lighthouse-result.proto

```diff
@@ -359,6 +359,75 @@ message I18n {
 
     // This label is for a filter checkbox above a table of items
     string third_party_resources_label = 20;
+
+    // Descriptive explanation for emulation setting when emulating a generic desktop form factor, as opposed to a mobile-device like form factor.
+    string runtime_desktop_emulation = 21;
```

Comment:
Can we deprecate all of this and replace with a `map<string, string>`?

Reply:
Yeah, that is for sure a TODO.